lib/barbeque_client/runner.rb

```diff
@@ -7,28 +7,63 @@ module BarbequeClient
   class Runner < Sinatra::Base
     set :port, ENV['PORT'] || 3003
 
+    class << self
+      # global variable
+      def messages
+        @messages ||= {}
+      end
+    end
+
     post '/v1/job_executions' do
```

Comment:
While I don't remember why I left this, this endpoint won't be called by barbeque_client and should be removed.